### ISSUE TITLE
Make a new coming soon page for resources page

### ISSUE THREAD

**Issue by rishipurwar1** [rishipurwar1/coding-space#39]
**Is your feature request related to a problem? Please describe.**
Make a new coming soon page for a resource page.

**Describe the solution you'd like**
You can add a coming soon illustration or do something creative to build a beautiful coming soon that matches the color scheme of our website.

If anyone wants to work on this issue, please let me know in the comment section.

**Comment by AyoubBoulahtar** [comment]
Hello, can I work on this ?

**Comment by rishipurwar1** [comment]
@AyoubBoulahtar  Yes! You can work on this issue.
Please share the photo of the page before committing your code.

**Comment by demonicirfan** [comment]
Hii, can i work on this i have some good design skills

**Comment by rishipurwar1** [comment]
Yes, you can work on this issue with @AyoubBoulahtar.
Please keep your discussion over here.

**Comment by rishipurwar1** [comment]
> Hii, can i work on this i have some good design skills

Yes, you can work on this issue with @AyoubBoulahtar.
Please keep your discussion over here.

**Comment by AyoubBoulahtar** [comment]
@demonicirfan let's see what you have in mind

**Comment by demonicirfan** [comment]
yes great i will create the graphics first and then share it here

**Comment by AyoubBoulahtar** [comment]
@demonicirfan When can you get something ready ?

**Comment by demonicirfan** [comment]
@AyoubBoulahtar @rishipurwar1 I made this can you review it, please
![MacBook - 1for review](https://user-images.githubusercontent.com/74172466/135315807-f949d74b-48eb-40cb-9bf0-9c3a353472b5.png)

**Comment by demonicirfan** [comment]
I dont know how to animate it if you will refer me to some resource that will help me to animate this then I will code that too

**Comment by AyoubBoulahtar** [comment]
It blends good with the overall design system, think about adding some sort of call to action, maybe a newsletter subscription field.

**Comment by demonicirfan** [comment]
![image](https://user-images.githubusercontent.com/74172466/135317086-9252b6ae-04d0-4b72-93a0-d7f1d1069a25.png)

**Comment by AyoubBoulahtar** [comment]
Take your time to enhance it further, but anyway, this is the right path

**Comment by demonicirfan** [comment]
Thanks what more should I improve but it will be there for some time and giving more effort will not be that much valuable I think. Instead, we should focus on other features. I just finished the mobile version.
![13 Pro - 1mobile](https://user-images.githubusercontent.com/74172466/135318283-cf7b63b1-a38b-4da8-a76a-bef426bcfada.png)

**Comment by rishipurwar1** [comment]
> I dont know how to animate it if you will refer me to some resource that will help me to animate this then I will code that too

You can use this tool to animate the SVG illustration.
https://www.svgator.com/

**Comment by demonicirfan** [comment]
ok I have started learning this tool and i will animate that svg can i start coding this page also

**Comment by rishipurwar1** [comment]
> ok I have started learning this tool and i will animate that svg can i start coding this page also

You need to discuss with @AyoubBoulahtar

**Comment by demonicirfan** [comment]
@AyoubBoulahtar hii man so whats your plan should we start it how to proceed I am learning how to animate that svg it will be done by tommorrow

**Comment by AyoubBoulahtar** [comment]
@demonicirfan I'm here whenever you have the design ready, we can implement it, just ping me when you're done, here or twitter.

**Comment by rishipurwar1** [comment]
> here or twitter

Please keep your discussion here only, so, more people can give their feedback.🙂

**Comment by rishipurwar1** [comment]
@AyoubBoulahtar @demonicirfan Feel free to star the repo

**Comment by demonicirfan** [comment]
![svgAnimation](https://user-images.githubusercontent.com/74172466/135609124-94164efe-c820-4af4-a809-4d36b21433a4.gif)

This is what i could manage to make with a basic trail account its smooth but not showing here properly as i have screen recorded it know what next

**Comment by AyoubBoulahtar** [comment]
@rishipurwar1 what svg loader are you using ?

**Comment by rishipurwar1** [comment]
> ![svgAnimation](https://user-images.githubusercontent.com/74172466/135609124-94164efe-c820-4af4-a809-4d36b21433a4.gif)
> 
> This is what i could manage to make with a basic trail account its smooth but not showing here properly as i have screen recorded it know what next

Is there a sudden jerk at the end of the animation?
Overall it looks great

**Comment by rishipurwar1** [comment]
> @rishipurwar1 what svg loader are you using ?

You can find loader in this path  /src/assets/animated_illustrations/loader.json

**Comment by demonicirfan** [comment]
no that is not jerk i screen recorded it that is why

**Comment by rishipurwar1** [comment]
> no that is not jerk i screen recorded it that is why

Ok!
Have you completed the design part?

**Comment by demonicirfan** [comment]
Yes i have completed it

**Comment by AyoubBoulahtar** [comment]
@demonicirfan do you need this PR for Hacktoberfest ? We can work on it together but you send the PR

**Comment by rishipurwar1** [comment]
Please share your design @demonicirfan

**Comment by demonicirfan** [comment]
yes @AyoubBoulahtar hacktoberfest is a motivation but I want to work on some front end open source projects too

**Comment by demonicirfan** [comment]
I have uploaded the files lets work together on this please guide me and how can we work on this please share the details too. I uploaded all the files and will share them [here](https://drive.google.com/drive/folders/1B2gB9U3VAK2J0CWkqrXQhUGr0KAwWEbh?usp=sharing)

**Comment by rishipurwar1** [comment]
I can't work right now together because of my ongoing lab exams. I have to study for that. But over this weekend I can help you.

**Comment by AyoubBoulahtar** [comment]
@demonicirfan dm me in Twitter

**Comment by demonicirfan** [comment]
ok i sent you a follow request

**Comment by rishipurwar1** [comment]
> I have started doing it

ok
